src/components/molecules/SpaceEditForm/SpaceEditForm.tsx

```diff
@@ -355,6 +364,13 @@ export const SpaceEditForm: React.FC<SpaceEditFormProps> = ({
                         {errors?.venue?.iframeUrl}
                       </span>
                     )}
+                    <h4 className="italic input-header">Enable autoplay</h4>
```

Comment:
```suggestion
```

The label of `AdminCheckbox` adds the necessary styles, additionally there is a prop for the label positoin (I recommend keeping it like in 
![sparkle-toggles-04](https://user-images.githubusercontent.com/79229621/140299088-0af1eca0-014a-4d45-a7fc-0246cc8b40bb.png)

Reply:
done 77985a4